test/common_plat/performance/odp_pktio_perf.c

```diff
@@ -788,6 +789,13 @@ static int test_init(void)
 		return -1;
 	}
 
+	/* Disable packet parsing as this is done in the driver where it
+	 * affects scalability.
+	 */
+	odp_pktio_config_init(&cfg);
+	cfg.parser.layer = ODP_PKTIO_PARSER_LAYER_NONE;
+	odp_pktio_config(gbl_args->pktio_rx, &cfg);
+
```

Comment:
The RX  side function pktio_pkt_has_magic() uses odp_packet_l4_offset(), so packet parsing cannot be simply disabled.

Reply:
Since these are synthetic packets that are constructed by this program. There's no need to call `odp_packet_l4_offset()`. This call can be replaced with `l4_off =  ODPH_IPV4HDR_LEN + ODPH_ETHHDR_LEN;` in `odp_packet_has_magic()`.